include/stdshims.h

```diff
@@ -7,8 +7,10 @@
 #include <memory>
 #include <utility>
 
-#if (__cplusplus >= 201402L)
+#if (__cplusplus >= 201402L || _MSC_VER)
```

Comment:
It looks VS 2012 incompatible.
https://github.com/LMMS/lmms/pull/4352#discussion_r191983580